app/src/main/java/io/intrepid/contest/models/Entry.java

```diff
@@ -1,12 +1,47 @@
 package io.intrepid.contest.models;
 
+import android.os.Parcel;
+import android.os.Parcelable;
+
 import java.util.Date;
 import java.util.UUID;
 
-public class Entry {
+public class Entry implements Parcelable {
```

Comment:
Rather than make this Parcelable, does it make sense to just request the information again from the server? That way we aren't caching the value to the app and it simplifies the amount of data being passed through intents.